Letters/src/test/java/org/Marta/LowercaseTextTest.java

```diff
@@ -0,0 +1,17 @@
+package org.Marta;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LowercaseTextTest {
+
+    @Test
+    void formatText() {
+        LowercaseText lowercaseText = new LowercaseText();
+        String text = "Future Collars";
+        String result = lowercaseText.formatText(text);
+        Assertions.assertEquals("future collars", result);
```

Comment:
struktura // given // when // then

+ nazwa testu raczej powinna wyglądać w taki sposób:

should[...]when(opcjonalnie)[...])

np.
shouldFormatText (już lepiej niż "formatText()")

testy sprawdzają jakieś zachowanie - sprawdzają, że kod "powinien" (should) coś zrobić ;)

Reply:
czemu nie "formatText()" ? Bo ten test nie formatuje tekstu. On **sprawdza** formatowanie